app/controllers/clearance/passwords_controller.rb

```diff
@@ -76,6 +75,13 @@ def find_user_for_update
     find_user_by_id_and_confirmation_token
   end
 
+  def ensure_existing_user
+    unless find_user_by_id_and_confirmation_token
+      flash_failure_when_forbidden
+      render template: 'passwords/new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.